doc/developer-guides/hld/virtio-blk.rst

```diff
@@ -65,6 +65,9 @@ The device model configuration command syntax for virtio-blk is::
 
    -s <slot>,virtio-blk,<filepath>[,options]
 
+- ``b``: needed when use vsbl as the virtual bootloader, to indicate the
```

Comment:
Can you provide an example use of this ``b`` parameter?  Where do you put the ``b`` (and its parameters) in the syntax description above?

Also, edit the wording to be something like:
```
- ``b``: when using ``vsbl`` as the virtual bootloader, this indicates the
  bootable device and the bootable image location. If used, it must be
  the first option.
```

Reply:
updated, please help to have a check.